src/attribution/onTTFB.ts

```diff
@@ -24,10 +24,22 @@ import {
 } from '../types.js';
 
 const attributeTTFB = (metric: TTFBMetric): void => {
-  if (metric.entries.length) {
+  // Only attribute if you have a non-zero responseStart value and a
+  // navigationEntry entry.
+  // E.g. For cross-origin redirects, Safari has a workerStart time but no
+  // TTFB time, so do not provide attribution for that.
+  if (metric.entries.length && metric.entries[0].responseStart) {
```

Comment:
I believe the logic here already includes a check for valid `responseStart` values.

Reply:
Oh good spot that the main metric checks this with `isInvalidTimestamp`. Removed in https://github.com/GoogleChrome/web-vitals/pull/458/commits/d2fb0aed6189985f1509886d3819961e34a3247e